client/state/ui/editor/media/imageEditor/actions.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Internal dependencies
+ */
+import {
+	IMAGE_EDITOR_ROTATE_COUNTERCLOCKWISE,
+	IMAGE_EDITOR_FLIP,
+	IMAGE_EDITOR_SET_FILE_INFO,
+	IMAGE_EDITOR_STATE_RESET
+} from 'state/action-types';
+
+export function resetImageEditorState() {
+	return {
+		type: IMAGE_EDITOR_STATE_RESET
+	};
+}
+
+export function imageEditorRotate() {
```

Comment:
Maybe `imageEditorRotateCounterclockwise` unless you'd like to pass through a direction param.